internal/pkg/deploy/pipeline.go

```diff
@@ -238,18 +240,23 @@ func PipelineSourceFromManifest(mfSource *manifest.Source) (source interface{},
 }
 
 // PipelineBuildFromManifest processes manifest info about the build project settings.
-func PipelineBuildFromManifest(mfBuild *manifest.Build) (build *Build) {
+func PipelineBuildFromManifest(mfBuild *manifest.Build, plName string) (build *Build) {
 	image := defaultPipelineBuildImage
 	environmentType := defaultPipelineEnvironmentType
+	path := fmt.Sprintf(fmtDefaultPipelineBuildspecPath, plName)
```

Comment:
Does it work if they have a pipeline manifest in the legacy path and want to deploy the pipeline?

Reply:
Fixed.